dictionaries/CallMap_74_delta.php

```diff
@@ -45,6 +45,10 @@
       'old' => ['resource|false', 'command'=>'string', 'descriptor_spec'=>'array', '&pipes'=>'resource[]', 'cwd='=>'?string', 'env_vars='=>'?array', 'options='=>'?array'],
       'new' => ['resource|false', 'command'=>'string|array', 'descriptor_spec'=>'array', '&pipes'=>'resource[]', 'cwd='=>'?string', 'env_vars='=>'?array', 'options='=>'?array'],
     ],
+    'strip_tags' => [
+      'old' => ['string', 'string'=>'string', 'allowed_tags='=>'?string'],
```

Comment:
As I understand it, null was not accepted prior to PHP 8.0

Reply:
I also found this information in documentation. But I also checked on PHP 7.2 - it also works with `null`: https://3v4l.org/nkHYY#v7.2.0

Reply:
Yeah, but that's because of undefined behaviour. Just because it used to work doesn't mean that it was designed like that or that it should be used this way. 
Arrays worked too https://3v4l.org/PE5hP#v7.2.0 and so did booleans https://3v4l.org/qoDFT#v7.2.0
Even with strict types, you could pass anything you'd like https://3v4l.org/mJvL3#v7.2.0 That's why in PHP 8.0 and 8.1 this behaviour was fixed.

Reply:
@kamil-tekiela 
thanks a lot, should be fixed by d33c338